aydin/nn/models/torch/torch_jinet.py

```diff
@@ -0,0 +1,291 @@
+from collections import OrderedDict
+from itertools import chain
+
+import torch
+from torch import nn
+from torch.optim.lr_scheduler import ReduceLROnPlateau
+from torch.utils.tensorboard import SummaryWriter
+
+from aydin.nn.layers.dilated_conv import DilatedConv
+from aydin.nn.pytorch.optimizers.esadam import ESAdam
```

Comment:
I just realize you are using a custom optimizer. Is it better than other existing optimizers like `AdamW` or `ranger21`?
Irrelevant to this code but the docstring in `ESAdam` ends in the middle. :D

Reply:
I am not sure which one is  better  @li-li-github , how to  benchmark different optimizers with respect to each other? just run n2s with different optimizers and check average convergence time?

Reply:
I would check the if any of these optimizers produce better images or lower losses but convergence time can be a good indicator too.

Reply:
have you had chance to look into this @li-li-github ? I will merge this as is but we can make a follow-up if you find anything interest